src/dev/mocha/run_mocha_cli.js

```diff
@@ -42,6 +42,8 @@ export function runMochaCli() {
 
   // check that we aren't leaking any globals
   process.argv.push('--check-leaks');
+  // prevent globals injected from canvas plugins from triggering leak check
+  process.argv.push('--globals', 'core,regeneratorRuntime,_');
```

Comment:
Kind of fragile, but at least future globals will break the CI. 

I do wonder, is there a way to remove that lodash global?